examples/FFNet.cpp

```diff
@@ -7,14 +7,15 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
-#include <afml/nn.h>
+#include <af/nn.h>
 
 using namespace af;
-using namespace afml;
-using namespace afml::nn;
+using namespace af;
```

Comment:
A tool would be great. Unfortunately, I'm just an irritating nitpicker. :innocent: